modules/apps/remote-app/remote-app-service/service.xml

```diff
@@ -39,7 +39,45 @@
 
 		<reference entity="User" package-path="com.liferay.portal" />
 	</entity>
+	<entity local-service="true" name="RemoteCustomElementEntry" remote-service="false" trash-enabled="false" uuid="true">
+
```

Comment:
For the future, I would try to be more enfatic justifying the need for another table... "because custom elements need an extra field" does not sound as a very important requirement.